Deprecated/AppController+Debug.swift

```diff
@@ -10,27 +10,8 @@ import Foundation
 import UIKit
 
 extension AppController {
-
     func showDebug() {
         let controller = UINavigationController(rootViewController: DebugViewController())
         self.present(controller, animated: true, completion: nil)
     }
-
-#if DEBUG
-    // swiftlint:disable override_in_extension
-    override var canBecomeFirstResponder: Bool {
-        true
-    }
-
-    override func motionEnded(_ motion: UIEvent.EventSubtype, with event: UIEvent?) {
-        guard motion == .motionShake else { return }
-        self.presentDebugViewController()
```

Comment:
I used this gesture to open the debug menu a lot. Can you explain why you removed it?

Reply:
It gets in the middle of the search bar and causes it to resign first responder as soon as you tap on it. I digged into it and it also was the cause of why the previous developer had to do this fix (even though he didn't know why he had the bug) in DirectoryViewController:

`
    // These two functions are implemented to avoid a bug where the initial
    // tap of the search bar begins editing, but first responder is immediately resigned
    // I can't figure out why this is happening, but this is a potential solution to avoid the bug.
    // I set a symbolic breakpoint and can't find why resignFirstResponder is being called there.
    //
    // first, when the edit begins, we store the date in self.searchEditBeginDate
    // then, in searchBarShouldEndEditing, we check whether this date was extremely recent
    // if it was too recent to be performed intentionally, we don't allow the field to end editing.
    func searchBarShouldBeginEditing(_ searchBar: UISearchBar) -> Bool {
        self.searchEditBeginDate = Date()
        return true
    }
`

The difference is that I couldn't, or I didn't know how, to do that sort of hack in swiftUI. I talked about this during standup and we agreed to just remove the shake to open the debug.

However, I do believe we can bring it up in the Onboarding.

Reply:
Oh weird. UIKit is so broken haha. I think I found a workaround, I will open a separate PR for it so it won't block this one.